### ISSUE TITLE
bug: vite warnings when minifying css

### ISSUE THREAD

**Issue by neapolis79** [saadeghi/daisyui#2731]
### What version of daisyUI are you using?

v4.5.0

### Which browsers are you seeing the problem on?

All browsers

### Reproduction URL

NA

### Describe your issue

During build, Vite throws several warnings during CSS minification. This just started to occur when I upgraded DaisyUI to v4.5.0.
Here are the warnings:
▲ [WARNING] Unexpected "*" [css-syntax-error]

    <stdin>:6863:30:
      6863 │     .btm-nav > .dark\:disabled*:hover {
           ╵                               ^

▲ [WARNING] Unexpected "*" [css-syntax-error]

    <stdin>:6881:28:
      6881 │   .btm-nav > .dark\:disabled* {
           ╵                             ^

**Comment by saadeghi** [comment]
Please share a reproduction link

**Comment by peter-mw** [comment]
Hi, i have the same problem. Did you found solution ?  

this is my `tailwind.config.js`


```
import preset from '../../../../vendor/filament/filament/tailwind.config.preset'

export default {
    presets: [preset],
    plugins: [
        require("@tailwindcss/typography"),
        require("daisyui")
    ],
    daisyui: {
       themes: ["light", "dark"],
       base: false,
       styled: true,
       utils: true,
        prefix: "",
        logs: true,
        themeRoot: ":root",
    },
    mode: 'jit',
    content: [
        './app/Filament/Admin/**/*.php',
        './resources/views/filament/admin/**/*.blade.php',
        './vendor/filament/**/*.blade.php',
    ]
}
```

**Comment by saadeghi** [comment]
Please share a reproduction link. How can I help without seeing the code?

**Comment by peter-mw** [comment]
Hi,

Here is how to reploduce 

clone the branch `daisy-ui-bug` from https://github.com/microweber/microweber/tree/daisy-ui-bug

After that 

Run 
 

```
composer install
npm install
```

After install run 

`npm run dev`


You can see the file config here: https://github.com/microweber/microweber/blob/daisy-ui-bug/resources/css/filament/admin/tailwind.config.js

**Comment by saadeghi** [comment]
```
composer install

Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
Your lock file does not contain a compatible set of packages. Please run composer update.

  Problem 1
    - lcobucci/clock is locked to version 2.3.0 and an update of this package was not requested.
    - lcobucci/clock 2.3.0 requires php ~8.1.0 || ~8.2.0 -> your php version (8.3.0) does not satisfy that requirement.
  Problem 2
    - lcobucci/clock 2.3.0 requires php ~8.1.0 || ~8.2.0 -> your php version (8.3.0) does not satisfy that requirement.
    - league/oauth2-server 8.5.4 requires lcobucci/clock ^2.2 || ^3.0 -> satisfiable by lcobucci/clock[2.3.0].
    - league/oauth2-server is locked to version 8.5.4 and an update of this package was not requested.

```
```
composer update 
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires lcobucci/clock 2.3.0 -> satisfiable by lcobucci/clock[2.3.0].
    - lcobucci/clock 2.3.0 requires php ~8.1.0 || ~8.2.0 -> your php version (8.3.0) does not satisfy that requirement.
  Problem 2
    - laravel/passport[dev-fix/install-command, dev-fixes-1728, v12.0.0, ..., 12.x-dev] require league/oauth2-server ^8.5.3 -> satisfiable by league/oauth2-server[8.5.3, 8.5.4].
    - laravel/passport 13.x-dev requires league/oauth2-server ^9.0 -> satisfiable by league/oauth2-server[9.0.0-RC1, 9.0.0].
    - laravel/passport[v11.8.5, ..., 11.x-dev] require illuminate/http ^9.0|^10.0 -> satisfiable by illuminate/http[v9.0.0-beta.1, ..., 9.x-dev, v10.0.0, ..., 10.x-dev].
    - laravel/passport[v0.1.0, ..., v0.2.4, v1.0.0] require illuminate/encryption ~5.3 -> satisfiable by illuminate/encryption[v5.3.0, ..., 5.8.x-dev].
    - laravel/passport[v10.1.1, ..., 10.x-dev, v11.0.0, ..., v11.8.4] require lcobucci/jwt ^3.4|^4.0 -> satisfiable by lcobucci/jwt[3.4.0, ..., 3.4.x-dev, 4.0.0-alpha1, ..., 4.4.x-dev].
    - lcobucci/jwt[4.3.0, ..., 4.4.x-dev] require lcobucci/clock ^2.0 || ^3.0 -> satisfiable by lcobucci/clock[2.3.0].
    - league/oauth2-server[8.5.2, ..., 8.5.4] require lcobucci/clock ^2.2 || ^3.0 -> satisfiable by lcobucci/clock[2.3.0].
    - lcobucci/jwt[4.0.0-beta1, ..., 4.2.x-dev] require lcobucci/clock ^2.0 -> satisfiable by lcobucci/clock[2.3.0].
    - league/oauth2-server[9.0.0-RC1, ..., 9.0.0] require lcobucci/clock ^2.3 || ^3.0 -> satisfiable by lcobucci/clock[2.3.0].
    - Only one of these can be installed: illuminate/encryption[dev-master, v5.3.0, ..., 5.8.x-dev, v6.0.0, ..., 6.x-dev, v7.0.0, ..., 7.x-dev, v8.0.0, ..., 8.x-dev, v9.0.0-beta.1, ..., 9.x-dev, v10.0.0, ..., 10.x-dev, v11.0.0, ..., 11.x-dev], laravel/framework[v11.0.8]. laravel/framework replaces illuminate/encryption and thus cannot coexist with it.
    - Only one of these can be installed: illuminate/http[dev-master, v5.0.0, ..., 5.8.x-dev, v6.0.0, ..., 6.x-dev, v7.0.0, ..., 7.x-dev, v8.0.0, ..., 8.x-dev, v9.0.0-beta.1, ..., 9.x-dev, v10.0.0, ..., 10.x-dev, v11.0.0, ..., 11.x-dev], laravel/framework[v11.0.8]. laravel/framework replaces illuminate/http and thus cannot coexist with it.
    - laravel/passport[v1.0.1, ..., 1.0.x-dev, v2.0.0, ..., 2.0.x-dev, v3.0.0, ..., 3.0.x-dev, v4.0.0, ..., 4.0.x-dev, v5.0.0, ..., 5.0.x-dev, v6.0.0, ..., 6.0.x-dev, v7.0.0, ..., v7.5.1, v9.4.0, ..., 9.x-dev, v10.1.0] require phpseclib/phpseclib ^2.0 -> found phpseclib/phpseclib[2.0.0, ..., 2.0.x-dev] but it conflicts with your root composer.json require (~3.0).
    - laravel/passport[v8.0.0, ..., 8.x-dev, v9.0.0, ..., v9.3.2] require php ^7.2 -> your php version (8.3.0) does not satisfy that requirement.
    - lcobucci/jwt[4.0.0-alpha2, ..., 4.0.0-alpha3] require php ^7.2 -> your php version (8.3.0) does not satisfy that requirement.
    - illuminate/encryption[v5.6.0, ..., 5.8.x-dev] require php ^7.1.3 -> your php version (8.3.0) does not satisfy that requirement.
    - laravel/passport[v10.0.0, ..., v10.0.1] require php ^7.3 -> your php version (8.3.0) does not satisfy that requirement.
    - lcobucci/jwt[3.3.0, ..., 3.4.x-dev] require php ^5.6 || ^7.0 -> your php version (8.3.0) does not satisfy that requirement.
    - lcobucci/jwt 4.0.0-alpha1 requires php ^7.1 -> your php version (8.3.0) does not satisfy that requirement.
    - lcobucci/clock 2.3.0 requires php ~8.1.0 || ~8.2.0 -> your php version (8.3.0) does not satisfy that requirement.
    - Root composer.json requires laravel/framework v11.0.8 -> satisfiable by laravel/framework[v11.0.8].
    - Root composer.json requires laravel/passport * -> satisfiable by laravel/passport[dev-fix/install-command, dev-fixes-1728, v0.1.0, ..., v0.2.4, v1.0.0, ..., 1.0.x-dev, v2.0.0, ..., 2.0.x-dev, v3.0.0, v3.0.1, v3.0.2, 3.0.x-dev, v4.0.0, ..., 4.0.x-dev, v5.0.0, ..., 5.0.x-dev, v6.0.0, ..., 6.0.x-dev, v7.0.0, ..., v7.5.1, v8.0.0, ..., 8.x-dev, v9.0.0, ..., 9.x-dev, v10.0.0, ..., 10.x-dev, v11.0.0, ..., 11.x-dev, v12.0.0, ..., 12.x-dev, 13.x-dev].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
```
```
composer update --with-all-dependencies
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires lcobucci/clock 2.3.0 -> satisfiable by lcobucci/clock[2.3.0].
    - lcobucci/clock 2.3.0 requires php ~8.1.0 || ~8.2.0 -> your php version (8.3.0) does not satisfy that requirement.
  Problem 2
    - laravel/passport[dev-fix/install-command, dev-fixes-1728, v12.0.0, ..., 12.x-dev] require league/oauth2-server ^8.5.3 -> satisfiable by league/oauth2-server[8.5.3, 8.5.4].
    - laravel/passport 13.x-dev requires league/oauth2-server ^9.0 -> satisfiable by league/oauth2-server[9.0.0-RC1, 9.0.0].
    - laravel/passport[v11.8.5, ..., 11.x-dev] require illuminate/http ^9.0|^10.0 -> satisfiable by illuminate/http[v9.0.0-beta.1, ..., 9.x-dev, v10.0.0, ..., 10.x-dev].
    - laravel/passport[v0.1.0, ..., v0.2.4, v1.0.0] require illuminate/encryption ~5.3 -> satisfiable by illuminate/encryption[v5.3.0, ..., 5.8.x-dev].
    - laravel/passport[v10.1.1, ..., 10.x-dev, v11.0.0, ..., v11.8.4] require lcobucci/jwt ^3.4|^4.0 -> satisfiable by lcobucci/jwt[3.4.0, ..., 3.4.x-dev, 4.0.0-alpha1, ..., 4.4.x-dev].
    - lcobucci/jwt[4.3.0, ..., 4.4.x-dev] require lcobucci/clock ^2.0 || ^3.0 -> satisfiable by lcobucci/clock[2.3.0].
    - league/oauth2-server[8.5.2, ..., 8.5.4] require lcobucci/clock ^2.2 || ^3.0 -> satisfiable by lcobucci/clock[2.3.0].
    - lcobucci/jwt[4.0.0-beta1, ..., 4.2.x-dev] require lcobucci/clock ^2.0 -> satisfiable by lcobucci/clock[2.3.0].
    - league/oauth2-server[9.0.0-RC1, ..., 9.0.0] require lcobucci/clock ^2.3 || ^3.0 -> satisfiable by lcobucci/clock[2.3.0].
    - Only one of these can be installed: illuminate/encryption[dev-master, v5.3.0, ..., 5.8.x-dev, v6.0.0, ..., 6.x-dev, v7.0.0, ..., 7.x-dev, v8.0.0, ..., 8.x-dev, v9.0.0-beta.1, ..., 9.x-dev, v10.0.0, ..., 10.x-dev, v11.0.0, ..., 11.x-dev], laravel/framework[v11.0.8]. laravel/framework replaces illuminate/encryption and thus cannot coexist with it.
    - Only one of these can be installed: illuminate/http[dev-master, v5.0.0, ..., 5.8.x-dev, v6.0.0, ..., 6.x-dev, v7.0.0, ..., 7.x-dev, v8.0.0, ..., 8.x-dev, v9.0.0-beta.1, ..., 9.x-dev, v10.0.0, ..., 10.x-dev, v11.0.0, ..., 11.x-dev], laravel/framework[v11.0.8]. laravel/framework replaces illuminate/http and thus cannot coexist with it.
    - laravel/passport[v1.0.1, ..., 1.0.x-dev, v2.0.0, ..., 2.0.x-dev, v3.0.0, ..., 3.0.x-dev, v4.0.0, ..., 4.0.x-dev, v5.0.0, ..., 5.0.x-dev, v6.0.0, ..., 6.0.x-dev, v7.0.0, ..., v7.5.1, v9.4.0, ..., 9.x-dev, v10.1.0] require phpseclib/phpseclib ^2.0 -> found phpseclib/phpseclib[2.0.0, ..., 2.0.x-dev] but it conflicts with your root composer.json require (~3.0).
    - laravel/passport[v8.0.0, ..., 8.x-dev, v9.0.0, ..., v9.3.2] require php ^7.2 -> your php version (8.3.0) does not satisfy that requirement.
    - lcobucci/jwt[4.0.0-alpha2, ..., 4.0.0-alpha3] require php ^7.2 -> your php version (8.3.0) does not satisfy that requirement.
    - illuminate/encryption[v5.6.0, ..., 5.8.x-dev] require php ^7.1.3 -> your php version (8.3.0) does not satisfy that requirement.
    - laravel/passport[v10.0.0, ..., v10.0.1] require php ^7.3 -> your php version (8.3.0) does not satisfy that requirement.
    - lcobucci/jwt[3.3.0, ..., 3.4.x-dev] require php ^5.6 || ^7.0 -> your php version (8.3.0) does not satisfy that requirement.
    - lcobucci/jwt 4.0.0-alpha1 requires php ^7.1 -> your php version (8.3.0) does not satisfy that requirement.
    - lcobucci/clock 2.3.0 requires php ~8.1.0 || ~8.2.0 -> your php version (8.3.0) does not satisfy that requirement.
    - Root composer.json requires laravel/framework v11.0.8 -> satisfiable by laravel/framework[v11.0.8].
    - Root composer.json requires laravel/passport * -> satisfiable by laravel/passport[dev-fix/install-command, dev-fixes-1728, v0.1.0, ..., v0.2.4, v1.0.0, ..., 1.0.x-dev, v2.0.0, ..., 2.0.x-dev, v3.0.0, v3.0.1, v3.0.2, 3.0.x-dev, v4.0.0, ..., 4.0.x-dev, v5.0.0, ..., 5.0.x-dev, v6.0.0, ..., 6.0.x-dev, v7.0.0, ..., v7.5.1, v8.0.0, ..., 8.x-dev, v9.0.0, ..., 9.x-dev, v10.0.0, ..., 10.x-dev, v11.0.0, ..., 11.x-dev, v12.0.0, ..., 12.x-dev, 13.x-dev].
```

🥲 Is there a way to see the issue without composer stuff? Like… a minimal reproduction link where I can run vite and see the vite error?

FYI: https://stackoverflow.com/help/minimal-reproducible-example

**Comment by peter-mw** [comment]
hi, please delete `composer.lock` and run `composer install`
